src/shell-integration/bash/ghostty.bash

```diff
@@ -95,6 +95,78 @@ if [[ "$GHOSTTY_SHELL_FEATURES" == *"sudo"* && -n "$TERMINFO" ]]; then
   }
 fi
 
+# SSH Integration
+if [[ "$GHOSTTY_SHELL_FEATURES" == *ssh-* ]]; then
+  ssh() {
+    builtin local ssh_term ssh_opts
+    ssh_term="xterm-256color"
+    ssh_opts=()
+
+    # Configure environment variables for remote session
+    if [[ "$GHOSTTY_SHELL_FEATURES" == *ssh-env* ]]; then
+      ssh_opts+=(-o "SetEnv COLORTERM=truecolor")
+      ssh_opts+=(-o "SendEnv TERM_PROGRAM TERM_PROGRAM_VERSION")
+    fi
+
+    # Install terminfo on remote host if needed
+    if [[ "$GHOSTTY_SHELL_FEATURES" == *ssh-terminfo* ]]; then
+      builtin local ssh_user ssh_hostname
+
+      while IFS=' ' read -r ssh_key ssh_value; do
+        case "$ssh_key" in
+          user) ssh_user="$ssh_value" ;;
+          hostname) ssh_hostname="$ssh_value" ;;
+        esac
+        [[ -n "$ssh_user" && -n "$ssh_hostname" ]] && break
+      done < <(builtin command ssh -G "$@" 2>/dev/null)
+
+      builtin local ssh_target="${ssh_user}@${ssh_hostname}"
+
+      if [[ -n "$ssh_hostname" ]]; then
```

Comment:
If we change this to test `ssh_target` here, then we can remove the `ssh_target` check below on line 150.